webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/index.tsx

```diff
@@ -42,13 +42,12 @@ type Props = {
     category: ChannelCategory;
 };
 
-const getUnreadsIdsForCategory = makeGetUnreadIdsForCategory();
```

Comment:
i am not sure about this one, Now it runs and create a new function of for underneath it

Reply:
If we create it at this level, every category will call the same selector with a different category, breaking the cache.

With this change, yes, we create more selectors, so there is more memory being consumed, but the selectors shouldn't run so often.